conformance/fhir-ig-davinci-hrex/CHANGELOG

```diff
@@ -6,16 +6,18 @@
 1. - Modified ig-r4.json to remove unsupported values from the parameters array, such as  `copyrightyear`, `releaselabel`.
 
 2. - Change targetProfile for us-core profile references to dependency on 3.1.1, so it's not conflicting with 4.0.0 (http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient|3.1.1)
```

Comment:
since the updated version of hrex references 4.0.0, it wouldn't be appropriate for us to change their references to 3.1.1.  instead i think we should ensure all their references are versioned and, if not, update those include `|4.0.0`, then update this comment here to reflect that

Reply:
Updated.

Reply:
I see the references are updated, but the comment here is now even more stale.

I suggest changing it to something like:
> Add the version suffix (`|4.0.0`) for StructureDefinition references to US Core resources to lock them to this version

And double-check the list of affected files.